packages/ilib-po/package.json

```diff
@@ -54,37 +45,25 @@
         "url": "git@github.com:iLib-js/ilib-mono.git"
     },
     "scripts": {
-        "build": "pnpm run build:prod",
-        "build:pkg": "echo '{\"type\": \"commonjs\"}' > lib/package.json",
-        "build:prod": "tsc --project tsconfig.json --outDir lib --module commonjs",
-        "build:dev": "tsc --project tsconfig.json --outDir lib --module commonjs --pretty --sourceMap",
-        "dist": "npm-run-all --npm-path pnpm doc types build:prod build:pkg; pnpm pack",
-        "test": "LANG=en_US.UTF8 NODE_OPTIONS=\"$NODE_OPTIONS --experimental-vm-modules\" jest  --testEnvironment node",
-        "test:watch": "pnpm test:jest --watch",
-        "debug": "LANG=en_US.UTF8 NODE_OPTIONS=\"$NODE_OPTIONS --experimental-vm-modules --inspect-brk\" jest -i  --testEnvironment node",
+        "build": "tsc --project tsconfig.build.json",
+        "types": "tsc",
+        "dist": "npm-run-all --npm-path pnpm build doc; pnpm pack",
+        "test": "jest",
+        "test:watch": "pnpm test --watch",
+        "debug": "node --inspect-brk node_modules/jest/bin/jest.js --runInBand",
         "clean": "git clean -f -d src test",
         "doc": "mkdir -p docs ; npm-run-all --npm-path pnpm doc:html doc:md",
         "doc:md": "mkdir -p docs ; typedoc --entryFileName index --plugin typedoc-plugin-markdown --skipErrorChecking --cleanOutputDir false --readme none --out docs src/index.ts",
-        "doc:html": "mkdir -p docs ; typedoc --out docs --skipErrorChecking --cleanOutputDir false src/index.ts",
-        "types": "tsc --declaration --emitDeclarationOnly -p ./tsconfig.json --outDir lib --module commonjs --outFile ./ilib-po.d.ts"
+        "doc:html": "mkdir -p docs ; typedoc --out docs --skipErrorChecking --cleanOutputDir false src/index.ts"
     },
     "devDependencies": {
-        "@jest/globals": "^29.7.0",
```

Comment:
Without the jest globals, my IDE complains about undefined jest functions like describe() or test()